composer.json

```diff
@@ -18,11 +18,6 @@
         "friendsofphp/php-cs-fixer": "^2.14",
         "sphpdox/sphpdox": "dev-master"
     },
-    "extra": {
```

Comment:
@sabbelasichon just introduced that. What's the purpose?

Reply:
oh, sorry that was a rebase of my task branch on branch 2.0

Reply:
I revented that commit so the squash and merge will not change composer.json